scripts/ensure-python.sh

```diff
@@ -7,6 +7,7 @@ set -eu
 PY_VERSION=3.8.9
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-    which "/opt/circleci/.pyenv/versions/$PY_VERSION/bin/python" || pyenv install "$PY_VERSION"
+		# Install version PY_VERSION, but skip if already installed.
+    which "/opt/circleci/.pyenv/versions/$PY_VERSION/bin/python" || pyenv install "$PY_VERSION" -s
```

Comment:
`-s` flag skips if `pyenv` version is already installed.
https://github.com/pyenv/pyenv/blob/master/COMMANDS.md#pyenv-install
>  -s/--skip-existing    Skip the installation if the version appears to be installed already